### PR TITLE
Add Ukraininan language

### PR DESCRIPTION
**Pull request by dmytro-ielkin**
Hi everyone,

Continued PR from #411 

I was didn't create a branch from the very beginning and was working in my master and therefore could not really rebase and make things clean. So I removed and re-created the repository.

Please review and let me know if it's OK.

### CODE REVIEW

**Comment by dmytro-ielkin** [comment]
@hazzik Rebased commit should be on master or in add-ukrainian-language branch?

File: release_notes.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ###In Development
 
 [Commits](https://github.com/MehdiK/Humanizer/compare/v1.35.0...master)
+ - [#399](https://github.com/MehdiK/Humanizer/pull/414): Add Ukraininan language
```
Comment (mexx) [inline release_notes.md]: Please change the PR number in link text to `414` from `399`?

**Comment by dmytro-ielkin** [comment]
So I was finally able to rebase and squash commits into one.

What I don't understand is why is there an empty commit called 'Merge branch 'add-ukrainian-language' of...' I didn't create it and just appeared there after I synced commit after rebase.

**Comment by hazzik** [comment]
you need to force push

**Comment by dmytro-ielkin** [comment]
Should I just run git push --force

**Comment by hazzik** [comment]
Yes, now everything "up to date", but it is complete mess.

**Comment by hazzik** [comment]
You may want to do following

```bash
git remote add upstream https://github.com/MehdiK/Humanizer.git
git checkout add-ukrainian-language
git reset a6cfc0d -- hard
git rebase upstream/master -i
```
in the dialog squash wanted commits, edit the commit message
```bash
git push origin add-ukrainian-language:add-ukrainian-language -f
```

Or like this:

```bash
git remote add upstream https://github.com/MehdiK/Humanizer.git
git merge upstream/master
git reset upstream/master
```
Commit all files again. With appropriate commit message
```bash
git push origin add-ukrainian-language:add-ukrainian-language -f
```

**Comment by hazzik** [comment]
You still merge with your master. Dont do this:) Try again.

Before you begin again:

```
git checkout master
git reset upstream/master --hard
```

**Comment by dmytro-ielkin** [comment]
My master and my branch are now restored to desired state.

So now I squash my commits into one commit, correct?

**Comment by hazzik** [comment]
@dmytro-ielkin yes, please

**Comment by dmytro-ielkin** [comment]
Did I do it right this time?

File: src/Humanizer.Tests/Localisation/uk-UA/DateHumanizeTests.cs [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        [InlineData(11, "через 11 секунд")]
+        [InlineData(19, "через 19 секунд")]
+        [InlineData(20, "через 20 секунд")]
+        [InlineData(21, "через 21 секунду")]
+        [InlineData(22, "через 22 секунди")]
+        [InlineData(23, "через 23 секунди")]
+        [InlineData(24, "через 24 секунди")]
+        [InlineData(25, "через 25 секунд")]
+        [InlineData(40, "через 40 секунд")]
+        public void SecondsFromNow(int seconds, string expected)
```
Comment (MehdiK) [inline src/Humanizer.Tests/Localisation/uk-UA/DateHumanizeTests.cs]: You have one complex language!! ;)

**Comment by MehdiK** [comment]
Thanks for the contribution. This is now available on nuget as v1.36.0.

**Comment by dmytro-ielkin** [comment]
Awesome. Thank you too for the creating the project in the first place.

**Pull request merged**